### PR TITLE
Adding my changes. Please review

### PR DESCRIPTION
**Pull request by rishishah77**

### CODE REVIEW

File: madlibs_list.txt [hunk 1]
```diff
@@ -8,12 +8,12 @@
...
-13. Material
-14. Activity
-15. Adverb
-16. Number
+11. Liquid - blood
+12. Adjective - comfortable
+13. Material - mushrooms
+14. Activity - reading
+15. Adverb - murder mysteries
+16. Number - 1000
```
Comment (yaminipreethik) [inline madlibs_list.txt:25]: 16th blank is already handled and value exists in main branch. Please modify accordingly.

**Comment by yaminipreethik** [comment]
Please resolve merge conflicts in order for the pull request to be merged.

**Comment by rishishah77** [comment]
WIll do that again.

**Comment by janakigrad** [comment]
Changes looks good

**Review by janakigrad — COMMENTED** [review]
Everything looks good

**Review by janakigrad — COMMENTED** [review]
Perfect!

**Pull request closed without merging**